Tests/TestingApps/SimpleApp/SimpleApp/Views/AppContentView.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  AppContentView.swift
+//  SimpleApp
+//
+//  Created by Nacho Soto on 7/13/23.
+//
+
+import RevenueCat
+import RevenueCatUI
+import SwiftUI
+
+struct AppContentView: View {
+
+    var customerInfo: CustomerInfo?
+
+    var body: some View {
+        ZStack {
+            VStack(spacing: 20) {
+                Text(verbatim: "Welcome to the Simple App")
+                    .font(.largeTitle)
+                    .multilineTextAlignment(.center)
+
+                if let info = self.customerInfo {
+                    Text(verbatim: "You're signed in: \(info.originalAppUserId)")
+                        .font(.callout)
+
+                    if let date = info.latestExpirationDate {
+                        Text(verbatim: "Your subscription expires: \(date.formatted())")
+                            .font(.caption)
+                    }
+                }
+            }
+            .padding()
+
+            Rectangle()
+                .foregroundStyle(.orange.gradient)
+                .opacity(0.2)
+                .frame(maxWidth: .infinity, maxHeight: .infinity)
+                .edgesIgnoringSafeArea(.all)
+        }
+    }
```

Comment:
This is this simple screen:
![simulator_screenshot_4F32B1F0-9F4A-4DBE-8821-0233B1540DEF](https://github.com/RevenueCat/purchases-ios/assets/685609/e7df8f69-edce-417d-8ace-f85e747ec2ef)